src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -307,11 +278,7 @@ data:
           METRICS: 1
 ```
 
-If you are using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3), this ConfigMap will already be mounted in the required container.
-
-<Callout variant="important">
-  For the Kubernetes Integration version 2, you will need to add an entry for this ConfigMap in the `volumes` and `volumeMounts` section of the DaemonSet's `spec`, to ensure all the files in the ConfigMap are mounted in `/etc/newrelic-infra/integrations.d/`.
-</Callout>
+If you are using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), this ConfigMap will already be mounted in the required container.
```

Comment:
```suggestion
If you're using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), this `ConfigMap` will already be mounted in the required container.
```